packages/backend/src/app/database/database-connection.ts

```diff
@@ -50,10 +51,10 @@ const getMigrations = () => {
     return [
       FlowAndFileProjectId1674788714498,
       initializeSchema1676238396411,
+      encryptCredentials1676505294811
     ];
   }
-
-  return [];
+  return [encryptCredentials1676505294811];
```

Comment:
what is the reason for running this migration in development?

Reply:
Data migration, there is no schema changes.